content/en/docs/tutorials/github.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Setup GitHub Actions"
```

Comment:
If we want this tutorial to be specific to `score-humanitec` I would also make that clear in the meta description e.g. "A tutorial on how to set up score-humanitec with GitHub Actions"